PlayFabClientSDK/source/ISerializerPlugin.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace PlayFab
+{
+    /// <summary>
+    /// Interface of any data serializer SDK plugin.
+    /// </summary>
+    public interface ISerializerPlugin: IPlayFabPlugin
```

Comment:
We have traditionally had a bit more required interface to our serializer.
https://github.com/PlayFab/CSharpSDK/blob/master/PlayFabClientSDK/source/Json/ISerializer.cs#L8
In particular, deserializing an object of unspecified type (IE, just object).
And also the second parameter of jsonSerializerStrategy, which allows to set and override a custom strategy for json serialization (IE, we can specify whatever other/alternate mechanism we want for enums and timestamps and such).

Unity uses the same:
https://github.com/PlayFab/UnitySDK/blob/master/Source/PlayFabSDK/Shared/Internal/ISerializer.cs#L8